### ISSUE TITLE
Clarification of ERA5 processing

### ISSUE THREAD

**Issue by robmarkcole** [nasaharvest/galileo#18]
I understand Galileo was pre-trained on https://developers.google.com/earth-engine/datasets/catalog/ECMWF_ERA5_MONTHLY - this codebase lists the variable as 
```
ERA5_BANDS = ["temperature_2m", "total_precipitation_sum"]
```
I want to check this is the `total_precipitation` band from GEE, and confirm if any post processing was applied (I note the `_sum`).

I also have been investigating the range of values expected for this feature, the GEE docs state a range 0 - 0.4, however the normalisation applied in Galileo is 0-0.3 - I assume this was calculated on the subset of data used in pretraining?

In fine tuning galileo I've used the data from https://cds.climate.copernicus.eu/datasets/reanalysis-era5-land-monthly-means which is supported as a data_source in rslearn, however the values of `total_precipitation` feature are quite different, for instance the product retrieved with:
```
dataset = "reanalysis-era5-land-monthly-means"
request = {
    "product_type": ["monthly_averaged_reanalysis"],
    "variable": ["total_precipitation"],
    "year": ["2025"],
    "month": ["07"],
    "time": ["00:00"],
    "data_format": "netcdf",
    "download_format": "unarchived",
    "area": "global"
}

client = cdsapi.Client()
client.retrieve(dataset, request).download()
```
Has a mean value of 0.0019m, which is far lower. Possibly additional processing is required? 

<img width="691" height="470" alt="Image" src="https://github.com/user-attachments/assets/2c5bfd57-bfeb-4aa4-8848-e201ca447885" />

**Comment by gabrieltseng** [comment]
Also:
- ERA5 shift value for temperature - is this just wrong?
- Precipitation sum - what is this?

**Comment by jleben** [comment]
I'm sharing what I found regarding precipitation:
- The dataset used in Galileo pretraining seems to be this: https://developers.google.com/earth-engine/datasets/catalog/ECMWF_ERA5_LAND_MONTHLY_AGGR#citations - the description of `total_precipitation_sum` is not clear on what the summation time period is.
- The page above links to this as the source: https://cds.climate.copernicus.eu/datasets/reanalysis-era5-land-monthly-means?tab=overview which says the following about "Total precipitation": "This variable is accumulated from the beginning of the forecast time to the end of the forecast step."
- The document above in turn points to this: https://confluence.ecmwf.int/display/CKB/ERA5-Land%3A+data+documentation There, it is also hard parse the info pertaining to the total precipitation variable, but it seems the longest accumulation period for any variable is 24h.
